### PR TITLE
Update docs for Android August release

### PR DESCRIPTION
**Pull request by dshokouhi**
Please only merge this once the play store release is live!

- Removes all beta tags for the android release
- Updates all notification commands to the new format along with keeping the old parameter table and mentioning that things changed in 2022.8

### CODE REVIEW

**Comment by jpelgrom** [comment]
On the notification commands page, you've added the sentence "If you have updated the android app to 2022.8 you must use `command` in place of `title` like the below example:" a few times, but no example follows (usually it's the title of the next command). 

Also seeing "android" a few times, try using an uppercase A 🙂

**Comment by dshokouhi** [comment]
> On the notification commands page, you've added the sentence "If you have updated the android app to 2022.8 you must use `command` in place of `title` like the below example:" a few times, but no example follows (usually it's the title of the next command).

Thats right I added the examples then I realized I was just duplicating the example directly above it so I removed them but forgot to remove the mention of "the example" lol



> Also seeing "android" a few times, try using an uppercase A 🙂

oops ill get those fixed

**Comment by jpelgrom** [comment]
2022.8.0 is now available from the Play Store

File: docs/notifications/commands.md [hunk 1]
```diff
@@ -264,9 +226,9 @@ automation:
       - service: notify.mobile_app_<your_device_id_here>
         data:
           message: "command_broadcast_intent"
-          title: "action"
           data:
-            channel: "package-name"
+            intent_package_name: "package-name"
+            intent_action: "action
```
Comment (jpelgrom) [inline docs/notifications/commands.md:231]: Missing a `"` at the end

**Review by jpelgrom — APPROVED** [review]

**Pull request merged**